fb-install-katello.bats

```diff
@@ -31,7 +31,7 @@ setup() {
 
 @test "run katello-deploy" {
   cd katello-deploy/
-  ./setup.rb --version $KATELLO_VERSION --installer-options "--foreman-admin-password admin"
+  ./setup.rb --version $KATELLO_VERSION --installer-options "--foreman-admin-password ${KATELLO_ADMIN_PASSWORD:-admin}"
```

Comment:
> Any particular reason you're using a different variable name here instead of reusing it?

Main motivation is all Katello related ones start with `KATELLO_`...

-- 
Later,
 Lukas #lzap Zapletal